### PR TITLE
[Bulk| Validations: Fix missing alt text issues 7

### PR DESCRIPTION
**Pull request by v-lkasturi**
Adding alternate texts to screenshots.

### CODE REVIEW

File: iis/publish/using-web-deploy/web-deploy-parameterization.md [hunk 1]
```diff
@@ -37,63 +37,63 @@ This article presents each of these steps in detail.
...
 
 Out of the box, the photo gallery application uses a file-based, SDF database. However, most real-world applications use a non-file based database, so you will convert (or "migrate") the SDF database into a Microsoft SQL Server database. If you don't have SQL Server, use Web Platform Installer to search for "SQL Server Express 2008 R2". Next, click **Add,** and then click **Install** to install it:
 
-[![](web-deploy-parameterization/_static/image15.png)](web-deploy-parameterization/_static/image13.png)
+[![Screenshot shows database search results highlighting S Q L Server Express 2 zero zero 8 R 2.](web-deploy-parameterization/_static/image15.png)](web-deploy-parameterization/_static/image13.png)
 
 During the application setup, remember the administrative username ("sa") and the password that you enter:
 
-[![](web-deploy-parameterization/_static/image19.png)](web-deploy-parameterization/_static/image17.png)
+[![Screenshot showing the Web Platform Installation window choosing the Mixed Mode Authentication option and specifying the password.](web-deploy-parameterization/_static/image19.png)](web-deploy-parameterization/_static/image17.png)
```
Comment (TimShererWithAquent) [inline iis/publish/using-web-deploy/web-deploy-parameterization.md:60]: Most read "Screenshot shows .  .  ." not "showing" - just FYI

File: iis/publish/using-web-deploy/web-deploy-parameterization.md [hunk 2]
```diff
@@ -37,63 +37,63 @@ This article presents each of these steps in detail.
...
 
 After migration completes, you can verify that the application still works by running the application in a web browser. In the top control ribbon, click **Run** to launch the application in a web browser:
 
-![](web-deploy-parameterization/_static/image29.png)
+![Screenshot shows a web browser window highlighting the Run navigation icon.](web-deploy-parameterization/_static/image29.png)
 
 This should bring up an application that looks similar to this:
 
-[![](web-deploy-parameterization/_static/image33.png)](web-deploy-parameterization/_static/image31.png)
+[![Screenshot shows the home page of the Gallery application.](web-deploy-parameterization/_static/image33.png)](web-deploy-parameterization/_static/image31.png)
```
Comment (TimShererWithAquent) [inline iis/publish/using-web-deploy/web-deploy-parameterization.md:88]: Is homepage two words? I can't find it in the writing guide, so I'm not sure.

**Comment by v-lkasturi** [comment]
@Rick-Anderson @mcleblanc 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

CC: @MonicaRush @snoviking

**Review by Rick-Anderson — APPROVED** [review]

File: iis/publish/using-web-deploy/web-deploy-parameterization.md [hunk 3]
```diff
@@ -238,27 +238,27 @@ Copy the long number and use it to replace the value in the feed XML that looks
...
 
-[![](web-deploy-parameterization/_static/image57.png)](web-deploy-parameterization/_static/image55.png)
+[![Screenshot shows a text field to add custom application feed.](web-deploy-parameterization/_static/image57.png)](web-deploy-parameterization/_static/image55.png)
 
 Finally, click **OK** to close the options dialog.
 
 If everything works, a new entry will be displayed in the Web Platform Installer:
 
-[![](web-deploy-parameterization/_static/image61.png)](web-deploy-parameterization/_static/image59.png)
+[![Screenshot shows the Web Platform Installer 3 point zero Sample Applications window highlighting Vacation Picture App.](web-deploy-parameterization/_static/image61.png)](web-deploy-parameterization/_static/image59.png)
```
Comment (LizCasey1) [inline iis/publish/using-web-deploy/web-deploy-parameterization.md:251]: Do you also want to spell out the number 3 here?

**Pull request merged**